src/databricks/labs/ucx/cli.py

```diff
@@ -657,7 +658,7 @@ def assign_owner_group(
     else:
         workspace_contexts = _get_workspace_contexts(w, a, run_as_collection)
 
-    owner_group = workspace_contexts[0].group_manager.pick_owner_group(prompts)
+    owner_group = AccountGroupLookup(workspace_contexts[0].workspace_client).pick_owner_group(prompts)
```

Comment:
Unrelated to your changes, but shouldn't it get the `owner_group` for each workspace context?